controller/comments.go

```diff
@@ -112,10 +113,16 @@ func (c *CommentsController) Delete(ctx *app.DeleteCommentsContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
-		if *identityID != cm.CreatedBy {
```

Comment:
done in https://github.com/almighty/almighty-core/pull/1377/commits/6e1ea960781470fc90508cc612c428092ce2f5f8